components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -5955,21 +6011,29 @@ Default: Set by build-namelist.
        group="ctl_nl" valid_values="" >
 Weighting of divergence component when applying hyperviscosity.
 If < 0, uses nu.
-Default: Set by build-namelist.
+Default: -1.
+</entry>
+
+<entry id="nu_s" type="real" category="se"
+       group="ctl_nl" valid_values="" >
+Hyperviscosity applied to dynamics scalars [m^4/s].
```

Comment:
... applied to temperature/pot. temperature? dp is a scalar, too.

Reply:
does this need to be changed?

Reply:
no, thanks